src/presentation/App.jsx

```diff
@@ -1,26 +1,13 @@
-import logo from './assets/logo.svg';
 import './App.css';
+import Calculator from '../components/Calculator';
+import CalculatorModule from '../modules/CalculatorModule';
 
 function App() {
   return (
     <div className="app">
```

Comment:
- [ ] `App.js` should be a class based component as this is a requirement kindly fix it.

Snapshot of requirements
![image](https://user-images.githubusercontent.com/81932478/188748058-409f73f6-e6c5-4873-83fa-53cd1c8b0aa0.png)
![Uploading image.png…]()